docs/configuration/yazi.md

```diff
@@ -322,14 +322,25 @@ An array of `[width, height]`, maximum image size (in pixels) for decoding a sin
 
 ## [plugin] {#plugin}
 
+### fetchers {#plugin.fetchers}
```

Comment:
Please verify this for correctness.

Reply:
Fetchers are not complete yet and will undergo further changes, which is why they haven't been documented so far. Please comment it out or add a warning to the user.

Reply:
Merged all your fixes and added a warning for the fetchers here https://github.com/yazi-rs/yazi-rs.github.io/pull/161/files#diff-c6ba6fcffd8baf7909debec16a3fdfed1eff5b2973516e130347b7b82c0cf0aaR327